plugins/endpoints/src/main/java/org/pytorch/serve/plugins/endpoint/ExecutionParameters.java

```diff
@@ -0,0 +1,80 @@
+package org.pytorch.serve.plugins.endpoint;
```

Comment:
It's not clear if this plugin is needed to move the plugins SDK?

Reply:
This is an example of how to define the plugins. This example shows how to define endpoints which can be readily used/supported by SM services.